crwatcher/watcher_test.go

```diff
@@ -99,12 +101,52 @@ func TestSetupHandlerAddFunc(t *testing.T) {
 	cw.handler.OnAdd(r)
 }
 
+// Test to ensure that if we are given opt in criteria we only call ResourceAdded for a resource with the specified opt
+// in criteria.
+func TestSetupHandlerAddFuncOptIn(t *testing.T) {
```

Comment:
This should use the name filter not OptIn for the test name and the documentation above updated as well.

Reply:
Done